contrib/inventory/puppetdb.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python2
```

Comment:
The shebang should be `#!/usr/bin/env python`, unless it's not possible for this script to run on python 3 due to dependencies lacking support for python 3. CI will catch this once the earlier mentioned python 3 syntax error is fixed.